src/index.html

```diff
@@ -8,6 +8,33 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="card" data-qa="card">
+      <div class="card__image-container">
+        <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="imac"
+        >
+      </div>
+
+      <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <span class="card__code">Product code: 195434</span>
```

Comment:
I'd recommend you to use `p`
```suggestion
      <p class="card__code">Product code: 195434</p>
```